lib/game_team_data.rb

```diff
@@ -34,7 +34,7 @@ def initialize()
   end
 
   def self.create_objects
-    table = CSV.parse(File.read('./data/dummy_file_game_teams.csv'), headers: true, converters: :numeric)
```

Comment:
I see we removed all of the converters due to the conflicts with the spec harness that we discussed.